README.md

```diff
@@ -1,3 +1,151 @@
+  <img src="https://img.shields.io/badge/prettier-1A2C34?style=for-the-badge&logo=prettier&logoColor=F7BA3E" />
+  <img src="https://img.shields.io/badge/eslint-3A33D1?style=for-the-badge&logo=eslint&logoColor=white" />
+</div>
+
+This is a repository to support collaboration on using LLMs in behavioral economics experiments. e.g. library of relevant UI components and a library for calling LLMs.
+
+[Instance deployed at EPFL](./EPFL.md)
+
+## Project Structure
+
+```bash
+├── .vscode    # VSCode configuration
+├── firestore  # Firebase Firestore rules
+│
+├── docs       # Documentation
+│
+├── firestore  # Firestore rules and indexes
+│
+├── functions  # Firebase Cloud Functions
+│   ├── lib           # Build output
+│   ├── node_modules
+│   └── src           # Cloud functions source code
+│       ├── endpoints   # Cloud functions endpoints
+│       ├── seeders     # Model factories
+│       ├── utils       # Utilities
+│       ├── validation  # TypeBox validation utilities
+│       ├── app.ts      # Firebase app initialization
+│       └── index.ts    # Cloud functions entrypoint
+│
+└── webapp     # Webapp frontend source code
+    ├── node_modules
+    └── src               # Frontend source code
+        ├── app           # Angular JS 17 app & components
+        ├── assets        # Static assets
+        ├── environments  # Environment configuration
+        └── lib           # API, types & utilities
+```
+
+## Firebase
+
+This project uses Firebase as its backend. The configuration can be found in the [`.firebaserc`](./.firebaserc) and [`firebase.json`](./firebase.json) files.
+
+Install the firebase cli tools with the following commands:
+
+```bash
+npm install -g firebase-tools
+firebase login  # Login to the Google account destined to manage the Firebase project
+```
+
+### Emulators
+
+In order to run offline and for development purposes, we use Java Firebase emulators.
+
+```bash
+export JAVA_TOOL_OPTIONS="-Xmx4g"  # Set your desired max RAM (here: 4GB)
+firebase emulators:start  # Start the emulators
+```
+
+You will then be able to access the following UIs:
+
+- Authentication UI: `http://localhost:4000/auth`
+- Emulator UI: `http://localhost:4000`
+- Firestore UI: `http://localhost:4000/firestore`
+- Cloud Functions UI: `http://localhost:4000/functions`
+
+### Firestore
+
+We use Firestore as our database. The rules are located in the [`firestore.rules`](./firestore.rules) file.
+Basically, the database cannot be externally accessed, and must be interacted with through cloud functions.
+
+A database prototype schema can be found [here on dbdiagrams.io](https://dbdiagram.io/d/Firebase-LLM-Mediation-660d473a03593b6b61123f24) (readonly, change it with your own if you take-on the project).
+
+You can seed the database with the default data by running the following command:
+
+```bash
+curl "http://127.0.0.1:5001/friendlychat-d6dc5/us-central1/seedDatabase?seeder_password=seeder_password"
```

Comment:
I would prefer to have a script that we run locally that talks to the DB for initialization, rather than a web-hook with a magic password. Generally we should lean on existing authentication protocols, instead of making our own with passwords. You can do this quite easily because the cloud sdk authenticates with the project, so you can have a locally run script that will do initalization and remove the webhook for seedDatabase.